ansible_rulebook/cli.py

```diff
@@ -219,6 +220,23 @@ def get_parser() -> argparse.ArgumentParser:
         default=settings.skip_audit_events,
         help="Don't send audit events to the server",
     )
+    parser.add_argument(
+        "--vault-password-file",
+        help="The file containing one ansible vault password",
+        default=os.environ.get("ANSIBLE_VAULT_PASSWORD_FILE", ""),
```

Comment:
Do we want to honor the original environment variable from ansible-core? I think it might be confusing because is the only variable name from ansible that would be supported, it might suggest to users that we support others. 
With or without `ANSIBLE_VAULT_PASSWORD_FILE` I think we should support as well `EDA_VAULT_PASSWORD_FILE`